Develop reporte

CrisDELGADO opened the pull request:
reporte_usuario (por el moment xD)

CrisDELGADO committed:
merge1

...

CrisDELGADO committed:
reporte_usuario

xxxx

makintoshEi reviewed (commented):
esta vacansisimo loco ! No hagas reportes de Especialidad, solo de Pacientes, Medicos, Eventos, Talleres e Historial de Pacientes

![image](https://cloud.githubusercontent.com/assets/11714838/21689568/c5bebf30-d33e-11e6-8f44-876e60276c2b.png)

Seria bueno que no se muestre lo que esta señalado

CrisDELGADO committed:
reporte.. todo

Solo falta el de historia.. en esa base no se entiende.. necesito un
ejemplo o algo para guiarme

CrisDELGADO committed:
htaccess .l.

.l.

CrisDELGADO committed:
Reporte 100%

Ya esta pilas...
